dhis-2/dhis-support/dhis-support-db-migration/src/main/java/org/hisp/dhis/db/migration/v36/V2_36_11__Migrate_sharings_to_jsonb.java

```diff
@@ -27,14 +27,10 @@
  */
 package org.hisp.dhis.db.migration.v36;
 
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 
-import org.flywaydb.core.api.migration.BaseJavaMigration;
-import org.flywaydb.core.api.migration.Context;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.flywaydb.core.api.migration.*;
+import org.slf4j.*;
 
 public class V2_36_11__Migrate_sharings_to_jsonb
```

Comment:
Don't this require a new "script" as the one with this ID might already have been applied to a database?

Reply:
@jbee  This is an attempt to fix a flyway script thats *not released to public yet*. 
So core devs would need to revert the database to before 2.36.11 script is executed. 
We of course can't do this if the script is released.